ChangeLog

```diff
@@ -49,6 +50,8 @@ ver. 0.9.2 (2014/XX/XXX) - wanna-be-released
        - empty or "basic" works as-is
        - "cymru" additionally prints (ASN, Country RIR) per banned IP
        - Requires dnspython or dnspython3
+   - New ignorecommand:
+     - apache-fakegooglebot.sh  Thanks Lee Clemens
```

Comment:
I don't think it is beneficial to list it here as well.  IMHO New filter entry is sufficient (And you could extend with information on this ignorecommand or just omit it)